.github/workflows/ci.yaml

```diff
@@ -59,32 +59,25 @@ jobs:
             allure_on_amd64: false
           - agent: 3.4.6  # renovate: juju-agent-pin-minor
             allure_on_amd64: true
-          - snap_channel: 3.6/stable
-            allure_on_amd64: false
         architecture:
           - amd64
         include:
           - juju:
               agent: 3.4.6  # renovate: juju-agent-pin-minor
               allure_on_amd64: true
             architecture: arm64
-          - juju:
-              snap_channel: 3.6/stable
-              allure_on_amd64: false
-            architecture: arm64
```

Comment:
Please create a backlog ticket for the next pulse IF this is not restored ASAP/this week.
As discussed, 3.6 is our default and should not be forgotten. Tnx!